osf/management/commands/backfill_domain_references.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+import re
+import logging
+import datetime
+import operator
+from functools import reduce
+
+from django.db.models import Q
+from django.apps import apps
+from django.core.management.base import BaseCommand
+from osf.external.spam.tasks import check_resource_for_domains
+from addons.wiki.models import WikiVersion
+
+
+logger = logging.getLogger(__name__)
+
+DOMAIN_MATCH_REGEX = re.compile(r'(?P<protocol>\w+://)?(?P<www>www\.)?(?P<domain>[\w-]+\.\w+)(?P<path>/\w*)?')
+DOMAIN_SEARCH_REGEX = r'(http://[^ \'}\[\]\~\(\)\/]+|https://[^ \'}\[\]\~\(\)\/]+)'
+
+
+def spawn_tasks_for_domain_references_backfill(model, query, get_guid, batch_size=None, dry_run=False):
+    for item in model.objects.filter(query)[:batch_size]:
```

Comment:
`batch_size` is not required; does `qs[:None]` behave properly?

Reply:
> qs[:None] behave properly?

yes it returns the whole list.